bundle-workflow/python/build_workflow/build_recorder.py

```diff
@@ -35,13 +35,14 @@ def write_manifest(self, folder):
             yaml.dump(output_manifest.to_dict(), file)
 
     class BuildManifestBuilder:
-        def __init__(self, build_id, name, version, arch):
+        def __init__(self, build_id, name, version, arch, snapshot):
             self.data = {}
             self.data['build'] = {}
             self.data['build']['id'] = build_id
             self.data['build']['name'] = name
             self.data['build']['version'] = str(version)
```

Comment:
Thoughts on also appending -SNAPSHOT to this version string if its a snapshot? 

This would have the benefit of automatically setting tar builds with -SNAPSHOT

Reply:
Added.